### PR TITLE
Call it like it is: Atmospheric Technician

### PR DESCRIPTION
**Pull request by Jovaniph**
Flip the job titles so that Atmospheric Technician is the primary Job Title.

🆑 Jovaniph
tweak: Atmospheric Technician is now the primary job title.
/ :cl:

### CODE REVIEW

**Comment by Jountax** [comment]
Hail atmosia!

**Comment by Citinited** [comment]
I have the feeling that this is going to break a bunch of stuff that relies on "Life Support Specialist" as the default title, such as loadouts. Can you make sure everything is in order?

**Comment by Jovaniph** [comment]
yeah I can do that...This is just the start I realized DX

**Comment by tigercat2000** [comment]
This will break fluff items, player titles, and player occupation preferences. Is it really worth it for a name change?

**Comment by taukausanake** [comment]
@tigercat2000 I think it is. How do you even SQL anyways? Might be nice to get a crash course into it with issues like this

**Comment by FalseIncarnate** [comment]
I really don't see this as necessary, and I play atmos more than most other jobs.

Granted, Life Support Technician was made when they were supposed to also be responsible for the grav gen (which frankly never required any attention they were able to support without an engineer's help for power anyways), but even with that responsibility once again in limbo, I don't think it's worth changing the name.

It would be like making Microbiologist the default title for Virologist; changes nothing except what everyone is used to and adds nothing new as a result.

**Comment by Jovaniph** [comment]
@FalseIncarnate, You are right. It is not necessary.

**Comment by Fox-McCloud** [comment]
Please re-open this. This is the original name; it was originally only changed because of a reverted changed made a long time ago.

File: _maps/map_files/MetaStation/MetaStation.v41A.II.dmm [hunk 1]
```diff
@@ -1050,7 +1050,7 @@
...
 "aul" = (/obj/structure/cable{d2 = 8; icon_state = "0-8"},/obj/machinery/power/terminal,/obj/machinery/light/small{dir = 4},/obj/item/device/radio/intercom{frequency = 1459; name = "Station Intercom (General)"; pixel_x = 29},/turf/simulated/floor/plating,/area/maintenance/auxsolarstarboard)
-"aum" = (/obj/machinery/door/airlock/maintenance{icon = 'icons/obj/doors/Doorint.dmi'; name = "Brig Emergency Storage"; req_access_txt = "63"},/obj/machinery/atmospherics/pipe/simple/hidden/supply{dir = 4},/obj/structure/disposalpipe/segment{dir = 4},/obj/structure/cable/yellow{d1 = 4; d2 = 8; icon_state = "4-8"},/turf/simulated/floor/plating,/area/security/brig)
+"aum" = (/obj/machinery/door/airlock/maintenance{icon = 'icons/obj/doors/doorint.dmi'; name = "Brig Emergency Storage"; req_access_txt = "63"},/obj/machinery/atmospherics/pipe/simple/hidden/supply{dir = 4},/obj/structure/disposalpipe/segment{dir = 4},/obj/structure/cable/yellow{d1 = 4; d2 = 8; icon_state = "4-8"},/turf/simulated/floor/plating,/area/security/brig)
```
Comment (SamHPurp) [inline _maps/map_files/MetaStation/MetaStation.v41A.II.dmm]: Your IDE is trying to break things. This is bad. Stop it.

Comment (Jovaniph) [reply _maps/map_files/MetaStation/MetaStation.v41A.II.dmm]: I doubled checked to make sure doorint.dmi is the right casing and it is.... Metastation has it wrong...This is correct.

Comment (Jovaniph) [reply _maps/map_files/MetaStation/MetaStation.v41A.II.dmm]: There was nothing I can do to fix it other than to manually change it in the file and i'm not doing it. I might as well leave it.

Comment (fludd12) [reply _maps/map_files/MetaStation/MetaStation.v41A.II.dmm]: Not to mention that capitals in paths has almost *never* been out code standard.

**Comment by Jovaniph** [comment]
Accidently deleted it like an idiot

**Comment by TullyBurnalot** [comment]
AVÉ ATMOSIA

**Comment by SamHPurp** [comment]
@Jovaniph You're going to need to have defined SQL queries for this to merge I suspect, or this'll wreak havoc in the database.

**Comment by Jovaniph** [comment]
@SamHPurp How will I do that exactly? I though the SQL stuff requires me to have access to the server or something.

**Comment by SamHPurp** [comment]
@Jovaniph I'd recommend setting up local sql for test purposes regardless. 

Checkout the SQL schema files in your root installation... But to give you an idea, there are three SQL queries we had to run for my coroner PR, which may be helpful to you. #7014

**Comment by Jovaniph** [comment]
I did it again..fuck me

**Comment by Jovaniph** [comment]
I tried using SQL. I don't know how to SQL. Someone take over this PR, thank you.

**Pull request closed without merging**